tests/screenshot/apps/components/SwitchItem.js

```diff
@@ -5,18 +5,34 @@ import {withConfig} from './utils';
 
 const SwitchItemTests = [
 	<SwitchItem />,
+	<SwitchItem>Hello SwitchItem</SwitchItem>,
+	<SwitchItem selected>Hello SwitchItem</SwitchItem>,
+	<SwitchItem disabled>Hello SwitchItem</SwitchItem>,
+	<SwitchItem disabled selected>Hello SwitchItem</SwitchItem>,
+	<SwitchItem inline>Hello SwitchItem</SwitchItem>,
+	<SwitchItem inline selected>Hello SwitchItem</SwitchItem>,
 
 	// Icon slotAfter
 	<SwitchItem>SwitchItem<Icon slot="slotAfter">home</Icon></SwitchItem>,
 
```

Comment:
Maybe you can also add a test with slotBefore and one with label.

Reply:
switchItem does not have `slotBefore` or `label`